amqp/rabbitmq.go

```diff
@@ -162,6 +162,24 @@ func (c *RabbitMQClient) Publish(routingKey string, payload []byte) error {
 	)
 }
 
+// PublishOnExchange takes care of publishing a message to a single RabbitMQ exchange.
+func (c *RabbitMQClient) PublishOnExchange(exchange string, payload []byte) error {
+	return c.Channel.Publish(
+		exchange,
+		"",
+		false, // mandatory
+		false, // immediate
+		aq.Publishing{
+			Headers:         aq.Table{},
+			ContentType:     "application/json",
+			ContentEncoding: "",
+			Body:            payload,
+			DeliveryMode:    aq.Persistent,
+			Priority:        0,
+		},
+	)
+}
+
```

Comment:
The `Publish` and `PublishOnExchange` functions are almost identical. Would be nice to make it a bit more reusable.

Reply:
Thanks @kristinn . Check latest commit for an updated version.

Reply:
Good stuff! Thanks!